packages/babel-types/src/definitions/core.ts

```diff
@@ -1992,7 +2009,7 @@ defineType("TaggedTemplateExpression", {
     quasi: {
       validate: assertNodeType("TemplateLiteral"),
     },
-    typeParameters: {
+    [process.env.BABEL_8_BREAKING ? "typeArguments" : "typeParameters"]: {
```

Comment:
Although this change also affect the flow AST, it does not have practical impact for flow codemod, because Foo does not support type arguments in TaggedTemplateExpression yet, e.g. `` foo<bar>`q` ``.